### ISSUE TITLE
"Error the port is not opened" should not keep on re-opening.

### ISSUE THREAD

**Issue by stuaxo** [utmapp/UTM#6005]
I closed my VM, and in the Qemu console window I get a popup "Error the port is not opened" if I close it, it pops up again.

It should probably pop-up only once - I guess if the user tries to interact with the console, they may(?) want it to pop up again, though it would may be better to have some other dismissible widget that doesn't block interaction.

**Comment by stuaxo** [comment]
Edit:  I was able to get rid of this error by dismissing it many times, but at the very least the error could do with some debouncing.

**Comment by osy** [comment]
Can you test on v4.5.x beta? I vaguely remember fixing something like this. If you still get it, can you provide details on reproducing the error and a screenshot is possible? Thanks

**Comment by stuaxo** [comment]
I'm not working at the place where I was using Linux in a VM for on a Mac via, Qemu and UTM and probably won't do that for quite a while again (I'm sure in a few years the issues I was having will be resolved).

Thanks for your help on this and various tickets, feel free to close this if it looks like I was the only one able to give feedback on it.
